documentation/tutorials.md

```diff
@@ -315,3 +315,45 @@ This tutorial walks you through the process of making a submission to a Kaggle c
     *   This command will show your submission, its status (e.g., `complete`, `error`), and your public/private scores if available.
 
 This completes the process of submitting to a Kaggle competition using the CLI!
+
+# Tutorial: How to Submit to a Code Competition
+
+This tutorial walks you through the process of submitting to a code competition on Kaggle.
+
+## Step 1: Find a Code Competition
+
+First, you need to find a code competition to participate in. You can browse the available competitions on the [Kaggle competitions page](https://www.kaggle.com/competitions).
+
+## Step 2: Download the Dataset
+
+Once you have chosen a competition, you need to download the dataset. You can do this using the `kaggle competitions download` command:
+
+```
+kaggle competitions download -c <competition-name>
+```
+
+Replace `<competition-name>` with the name of the competition you want to participate in.
+
+## Step 3: Create a Kernel
+
+Next, you need to create a kernel to work on your submission. A kernel is a computational environment that you can use to write and run your code. You can create a new kernel on the Kaggle website or by using the `kaggle kernels init` command.
+
+## Step 4: Write Your Code
+
+Now it's time to write your code! You can use any programming language or framework that you are comfortable with. The goal is to create a model that can make predictions on the test set.
```

Comment:
This feels misleading.  Yeah, they can use whatever they want to create the model, but they still have to do inference in python or R.  I think this tutorial should either say it has to be python/r or (more complicated) explain how to upload a model and write an inference notebook.

Regardless, it seems to be missing the step where it pushes a kernel, which is required for submission.

Reply:
Oops, sorry. I intended to change that and forgot.

You have to push the kernel before submitting? I thought it would get run when submitted.